tests/fixtures/unformatted.txt

```diff
@@ -1,2 +1,3 @@
 a = 'hello'
-b =  42
+b = ["a", "very", "very", "very", "very", "very", "very", "very", "very", "long", "line"]
```

Comment:
Not that long but long enough for black :) I added this line in the middle of the script to make sure we can format a selection in the middle of the text, that results in more lines in the final text.